runtime/src/lib.rs

```diff
@@ -457,17 +459,7 @@ pub struct PoolManager;
 
 impl LiquidityPoolManager<LiquidityPoolId> for PoolManager {
 	fn can_remove(pool_id: LiquidityPoolId) -> bool {
-		let (_, feur) = SyntheticTokens::get_position(pool_id, CurrencyId::FEUR);
-		if feur > 0 {
-			return false;
-		}
-
-		let (_, fjpy) = SyntheticTokens::get_position(pool_id, CurrencyId::FJPY);
-		if fjpy > 0 {
-			return false;
-		}
-
-		true
+		SyntheticTokens::can_remove(pool_id)
```

Comment:
I mean have synthetic tokens module implements `LiquidityPoolManager` so we can remove `struct PoolManager` here and have `type PoolManager = SyntheticTokens;`